apps/design/shared/src/layout.ts

```diff
@@ -1348,7 +1428,9 @@ function layOutBallotHelper(
           m,
         });
         if (height > gridHeight(m.MAX_CONTEST_ROW_HEIGHT, m)) {
-          throw new Error(`Contest ${contest.id} is too tall to fit on a page`);
+          throw new Error(
+            `Contest is too tall to fit on a page: ${contest.title} `
+          );
```

Comment:
longer-term I wonder if we could automatically attempt different densities to try to fit the large contests

Reply:
Something in that direction could be cool!